src/index.ts

```diff
@@ -37,17 +37,17 @@ export default function supernova(env: Galaxy) {
       const rootElement = useElement();
       const reactRoot = useReactRoot(rootElement);
       const layout = useStaleLayout() as TableLayout;
-      const app = useApp();
-      const model = useModel();
+      const app = useApp(); // undefined when taking snapshot
```

Comment:
These are all undefined when taking snapshot in nebula dev tool right? So not in Sense?

Reply:
that's correct

Reply:
Consider stating that then

Reply:
These are undefined all the way for in snapshot, so i do not think they should be stating